### PR TITLE
Add CORS flag, fix default descriptions, move server flag descriptions to proper section

### PR DESCRIPTION
**Pull request by ZeljkoBenovic**
* add access-control-allow-origins flag
* add more description for jsonrpc flag
* add additional desc for prometheus flag
* move flag desc to appropriate sections
*  minor fixes

Fixes EDGE-436

### CODE REVIEW

**Review by zivkovicmilos — APPROVED** [review]
Looks great 💯 

Thank you for fixing the docs in places that slipped through in the original reviews 🙏

File: docs/get-started/cli-commands.mdx [hunk 1]
```diff
@@ -494,66 +499,84 @@ Set block production time in seconds. Default: `2`
...
 
   </TabItem>
 </Tabs>
 
-Sets the target block gas limit for the chain. Default: `8000000`.
-
-A more detailed explanation on the block gas target can be found in the [TxPool section](/docs/architecture/modules/txpool#block-gas-target).
+Sets the premined accounts and balances in the format `address:amount`.
+The amount can be in either decimal or hex.
+Default premined balance: `0x3635C9ADC5DEA00000`.
```
Comment (Aleksao998) [inline docs/get-started/cli-commands.mdx:579]: I would like to note that this is the value in WEI -> ETH proportion, so for 1000 you need 1000000000000000000000. What do think?

**Review by Aleksao998 — APPROVED** [review]
Great work, thank you so much on going in depth and fixing all issues!

File: docs/get-started/cli-commands.mdx [hunk 2]
```diff
@@ -494,66 +499,84 @@ Set block production time in seconds. Default: `2`
...
-    dev --premine 0x3956E90e632AEbBF34DEB49b71c28A83Bc029862:1000000000000000000000
+    dev --log-level DEBUG
 
   </TabItem>
 </Tabs>
 
-Sets the premined accounts and balances in the format `address:amount`.
-The amount can be in either decimal or hex.
-Default premined balance: `0x3635C9ADC5DEA00000`.
+Sets the log level for console output. Default: `INFO`.
```
[the remark this answers is not in the document]
Comment (ZeljkoBenovic) [reply docs/get-started/cli-commands.mdx]: Added in bd790b88243930ab6def0740d729a07387349e20

File: docs/get-started/cli-commands.mdx (hunk 1 above)
Comment (ZeljkoBenovic) [reply docs/get-started/cli-commands.mdx:581]: Added in bd790b88243930ab6def0740d729a07387349e20

**Pull request merged**